pkg/cmd/get/get_build_logs_test.go

```diff
@@ -0,0 +1,69 @@
+package get_test
```

Comment:
I'd use package `get`. I think it is fully legit to test none exported functions.

Reply:
Will do!